components/IssueBadge/issueBadge.js

```diff
@@ -56,8 +56,12 @@ function BadgeList ({ badgeList, setBadgeChosen }) {
 }
 
 const setAvailableBadgeData = async (setBadgeList) => {
-  const badgeList = await callApi('badges')
-  setBadgeList(badgeList)
+  try {
+    const badgeList = await callApi('badges')
+    setBadgeList(badgeList)
+  } catch (e) {
+    console.error('Either Badgr server is down or you have not set BADGR credentials in environment', e)
```

Comment:
What's the impact on the user when this catch is triggered? Will they still be able to use the `IssueBadgeButton` or do we need to display an error message of some kind so that they're aware something is broken?

Reply:
The result will be there are no badges to issue.  An error would be nice but as this is 99% a dev issue the console log should suffice.

Reply:
OK makes sense :+1: